bottles/backend/managers/manager.py

```diff
@@ -374,12 +374,30 @@ def check_runners(self, install_latest: bool = True) -> bool:
                 shell=True
             ).communicate()[0].decode("utf-8")
             version = "sys-" + version.split("\n")[0].split(" ")[0]
-            self.runners_available.append(version)
+            runners_available.append(version)
 
         # check bottles runners
         for runner in runners:
             _runner = os.path.basename(os.path.normpath(runner))
-            self.runners_available.append(_runner)
+            runners_available.append(_runner)
+
+        runners_available = sorted(runners_available, reverse=True)
+
+        runners_order = {
```

Comment:
This breaks Wine-GE and Proton-GE runners. A fix would maybe to have an `other` category before the `sys-`, in which everything that is not sorted go in (sorted alphabetically)